packages/inter-protocol/test/auction/README-bidding-cli.md

```diff
@@ -0,0 +1,275 @@
+
+**Note:** The URLs of the faucet, RPC node, explorer, etc. are available at https://devnet.agoric.net/ .
+
+## Provisioning your account for use with Zoe smart contracts
+
+Introducing your account to Zoe involves provisioning an offer handler.
+
+Again, use https://devnet.agoric.net/ but this time, choose "send IST and provision ...".
+
+_TODO: cite inter.trade docs on provisioning a smart wallet?_
+
+
+## Liquidation Status
+
+To see what's available:
+
+```
+$ inter liquidation status
+{
+  "liquidatingCollateral": "116 ATOM",
+  "liquidatingDebt": "577.8142 IST",
+  "price": "12 IST/ATOM"
+}
+```
+
+## Choosing a network configuration with `AGORIC_NET`
+
+By default, `agops` tries to contact a local node. If you don't have one running, you may see...
+
+```
+$ inter liquidation status
+(Error#1)
+Error#1: cannot read data of published.agoricNames.brand: request to http://0.0.0.0:26657/abci_query?path=%22/custom/vstorage/data/published.agoricNames.brand%22&height=0 failed, reason: connect ECONNREFUSED 0.0.0.0:26657
+```
+
+https://devnet.agoric.net/network-config has details such as `chainName`, `rpcAddrs`. To tell `agops` to use it:
+
+```sh
+export AGORIC_NET=devnet
+```
+
+
+## Smart Contract Token Metadata: `inter vbank list`
+
+You may notice that `inter liquidation status` used the name `IST` where `agd query bank balances` shows `uist`. And the decimals were different.
+
+Electronic Rights Transfer Protocol (ERTP) is the token abstraction used in the Agoric smart contract platform. Cosmos-SDK denoms are reflected as ERTP issuers and brands via the **vbank**. Use `inter vbank list` To see the correspondence:
+
+```
+$ inter vbank list
+[
+  {
+    "issuerName": "BLD",
+    "denom": "ubld",
+    "brand": { "boardId": "board0223" },
+    "displayInfo": { "decimalPlaces": 6 }
+  },
+  {
+    "issuerName": "IST",
+    "denom": "uist",
+    "brand": { "boardId": "board0566" },
+    "displayInfo": { "decimalPlaces": 6 }
+  },
+  {
+    "issuerName": "ATOM",
+    "denom": "ibc/toyatom",
+    "brand": { "boardId": "board03446" },
+    "displayInfo": { "decimalPlaces": 6 }
+  }
+]
+```
+
+
+## Placing a bid: `inter bid ...`
+
+Let's place a 5 IST bid at a price of 9.50:
+
+```
+$ inter bid by-price --give 5IST --price 9.50 --from test-acct
+2023-03-31T07:07:33.319399551Z tx not in block 55762 retrying...
+2023-03-31T07:07:33.319399551Z tx not in block 55762 retrying...
+bid is broadcast:
+{"timestamp":"2023-03-31T07:07:38Z","height":"55763","offerId":"bid-1680246460034","txhash":"FDB98EEC71E987FEEE4A..."}
+{"id":"bid-1680246460034","price":"9.5 IST/ATOM","give":{"Currency":"5 IST"},"result":"Your bid has been accepted"}
+```
+
+This command does several things:
+
+  1. formats the bid as a Zoe offer specification
+  2. signs and broadcasts a transaction containing the offer spec
+  3. waits for the transaction to be included in a block
+  4. waits for the offer to be placed with the auctioneer contract via Zoe
+
+TODO: `inter bid by-discount`.
```

Comment:
Not needed (obviated by user guide which expands on usage and different bid types)